tests/foreman/ui/test_host.py

```diff
@@ -1714,3 +1714,25 @@ def test_positive_tracer_enable_reload(tracer_install_host, target_sat):
         )
         tracer = session.host_new.get_tracer(tracer_install_host.hostname)
         assert tracer['title'] == "No applications to restart"
+
+
+@pytest.mark.tier3
+@pytest.mark.rhel_ver_match('8')
+def test_positive_page_redirect_after_update(target_sat, current_sat_location):
+    """Check that page redirects correctly after editing a host without making any changes.
+    :id: 29c3397e-0010-11ef-bca4-000c2989e153
+    :steps:
```

Comment:
I suppose the quality checks are complaining about missing spaces between doscstring items, at least that is the difference compared to the original PR